### PR TITLE
Added the possibility to define the order of an export.

### PR DESCRIPTION
**Pull request by WarrenFaith**
### PR's key points
Added support to configure an order for the export.
 
### How to review this PR?
Carefully please, I have never worked on a plugin project before. I am also not sure if there would be a meaningful test to be written.
 
### Definition of Done
- [x] Changes summary added to CHANGELOG.md
- [x] Documentation added to README.md (if a new feature is added)
- [x] Tests added (if new code is added)
- [x] There is no outcommented or debug code left

PS: would be cool to have this supported for the swift/iOS version too, but I am not capable of doing that...

### CODE REVIEW

File: src/main/kotlin/com/hyperdevs/poeditor/gradle/Main.kt [hunk 1]
```diff
@@ -38,6 +38,7 @@ fun main() {
         ?.split(",")
         ?.map { it.trim() }
         ?: emptyList()
+    val order = dotenv.get("ORDER", "terms")
```
Comment (adriangl) [inline src/main/kotlin/com/hyperdevs/poeditor/gradle/Main.kt:41]: If you apply the suggestions I made, I'd set the value to `none`

File: src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorStringsImporter.kt [hunk 2]
```diff
@@ -86,6 +86,7 @@ object PoEditorStringsImporter {
                               defaultLang: String,
                               resDirPath: String,
                               filters: List<String>,
+                              order: String?,
```
Comment (adriangl) [inline src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorStringsImporter.kt:89]: If you apply the suggested changes, then make this a `String` value

File: src/main/kotlin/com/hyperdevs/poeditor/gradle/network/PoEditorApiController.kt [hunk 3]
```diff
@@ -56,17 +58,20 @@ class PoEditorApiControllerImpl(private val apiToken: String,
...
                                        filters: List<String>?,
+                                       order: String?,
                                        tags: List<String>?): String {
         val response = poEditorApi.getExportFileInfo(
             apiToken = apiToken,
             id = projectId,
             type = type.toString().toLowerCase(),
             filters = filters,
             language = code,
+            order = order,
```
Comment (adriangl) [inline src/main/kotlin/com/hyperdevs/poeditor/gradle/network/PoEditorApiController.kt:74]: If you use the enum approach, I would do something like this so we can pass the `NONE` value as null to the API:
```
            order = order.takeIf { it != OrderType.NONE }?.toLowerCase(), 
```

**Review by adriangl — CHANGES_REQUESTED** [review]
Hello! Sorry for the late reply, and thanks for the contribution!

Please review the changes I requested, they are mostly style and syntax suggestions, the code overall looks good apart from these changes.

**Comment by WarrenFaith** [comment]
Hey, thanks for the feedback. I rebased to latest master and took the FilterType as a guidance.
Let me know if there is still something I should change.

**Pull request merged**